string type         # Style of management: ["frame", "data", "child"]
string target_frame # Unity GameObject "frame" hosting the message 
string component    # UnityEngine.Transform
string modifier     # Unity function name to modify the scene